Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -1604,6 +1604,236 @@ void merge_reversible_connected_components(PolygonMesh& pm,
   }
 }
 
+
+/*!
+ * \ingroup PMP_orientation_grp
+ *
+ * identifies possible connections between the connected components of `pm` and if re-orientable,
+ * assigns a bit (`false` or `true`) to each face such that
+ * two faces with the same bits have a compatible orientation, while
+ * two faces with opposite bits have an incompatible orientation.
+ *
+ * @tparam PolygonMesh a model of `HalfedgeListGraph`, `FaceGraph`.
+ * @tparam FaceBitMap a model of `WritablePropertyMap` with `face_descriptor` as key and `bool` as value_type
+ * @tparam NamedParameters a sequence of \ref bgl_namedparameters
+ *
+ * @param pm a surface mesh
+ * @param fbm face bit map indicating if a face orientation should be reversed to be stitchable
+ *        (see `CGAL::Polygon_mesh_processing::stitch_borders()`) with another face. If `false` is
+ *        returned, the map will not be filled.
+ * @param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+ *
+ * @return `true` if `pm` can be reoriented and `false` otherwise.
+ *
+ * \cgalNamedParamsBegin
+ *   \cgalParamNBegin{vertex_point_map}
+ *     \cgalParamDescription{a property map associating points to the vertices of `pm`}
+ *     \cgalParamType{a class model of `ReadablePropertyMap` with `boost::graph_traits<PolygonMesh>::%vertex_descriptor`
+ *                    as key type and `%Point_3` as value type}
+ *     \cgalParamDefault{`boost::get(CGAL::vertex_point, pm)`}
+ *     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t`
+ *                     should be available for the vertices of `pm`.}
+ *   \cgalParamNEnd
+ * \cgalNamedParamsEnd
+ */
+template <class PolygonMesh, class FaceBitMap, class NamedParameters = parameters::Default_named_parameters>
+bool connected_components_compatible_orientations(PolygonMesh& pm,
+                                                  FaceBitMap fbm,
+                                                  const NamedParameters& np = parameters::default_values())
```

Comment:
The documentation should explain why `pm` is take by reference to non-const. When I read the documentation, I understand that the mesh is not modified (only property maps are), but when I read the signature of the function template, then I have a doubt.